src/components/AddBook.js

```diff
@@ -1,13 +1,42 @@
 import React from 'react';
+import { useDispatch } from 'react-redux';
+import { v4 as uuidv4 } from 'uuid';
+import { addBook } from '../redux/books/books';
 
-const Form = () => (
-  <form>
-    <input type="text" placeholder="Book title" />
-    <select disabled>
-      <option value="category">Category</option>
-    </select>
-    <button type="button">ADD BOOK</button>
-  </form>
-);
+const Form = () => {
```

Comment:
- [ ] This component should count with a local state with `title` and `author`, and the respective input should update the state every time the input change (`onChange`), kindly implement this 😄